### PR TITLE
Allowing frog to move with arrow keys in FoodChain activity

### PR DESCRIPTION
**Pull request by ricknjacky**
I did the necessary changes @llaske requested to be made

### CODE REVIEW

File: activities/FoodChain.activity/playgame.js [hunk 1]
```diff
@@ -10,14 +10,25 @@ FoodChain.playLevels = [
...
 FoodChain.playKeys = [
-	{ key: 75, heading: 0, dx: 1, dy: 0 },
-	{ key: 107, heading: 0, dx: 1, dy: 0 },
-	{ key: 73, heading: 90, dx: 0, dy: -1 },
-	{ key: 105, heading: 90, dx: 0, dy: -1 },
-	{ key: 74, heading: 0, dx: 1, dy: 0 },
-	{ key: 106, heading: 180, dx: -1, dy: 0 },
-	{ key: 76, heading: 270, dx: 0, dy: 1 },
-	{ key: 108, heading: 270, dx: 0, dy: 1 }
+	//commenting out the K key-codes and replacing
```
Comment (prakashujjwal1010) [inline activities/FoodChain.activity/playgame.js:13]: @ricknjacky , you don't need to add these comments.

**Comment by prakashujjwal1010** [comment]
Also, I have tested new changes at my end, it doesn't seem to work. I couldn't able to move frog by arrows keys.

File: activities/FoodChain.activity/playgame.js (hunk 1 above)
Comment (ricknjacky) [reply activities/FoodChain.activity/playgame.js:13]: @prakashujjwal1010 I added them, thinking some time in the future they might help.I will remove the comments and commit again :) thanks for the suggestion

**Comment by ricknjacky** [comment]
Actually i am new to the codebase, can u help me in here? Where did I make a mistake?

**Comment by llaske** [comment]
Your fix works. Thanks.
BTW you should not send PR on the master branch. See https://github.com/llaske/sugarizer/blob/dev/CONTRIBUTING.md
I've merged it manually in dev branch here https://github.com/llaske/sugarizer/commit/9630d2973641b13e86db7992c97fe3b794f08257

**Comment by ricknjacky** [comment]
point noted @llaske , won't do the same mistake again :)

**Pull request closed without merging**